toucan_connectors/workday/workday_connector.py

```diff
@@ -0,0 +1,153 @@
+import pandas as pd
+import workday
+from toucan_connectors.toucan_connector import ToucanConnector, ToucanDataSource
+from toucan_connectors.common import FilterSchema, transform_with_jq
+from pydantic import Field
+
+""" Only supports Anonymous or Credentials authentification for the moment """
+""" TODO: implement WS-Security X509-only signed credentials (Recommended by Workday) """
+from workday.auth import WsSecurityCredentialAuthentication
+from workday.auth import AnonymousAuthentication
+
+from datetime import datetime, timedelta, date
+import json
+import zeep
+import decimal
+import types
+
+
+def json_serial(obj):
+    """JSON serializer for objects not serializable by default json code"""
+
+    if isinstance(obj, (datetime, date)):
+        return obj.isoformat()
+    if isinstance(obj, decimal.Decimal):
+        return (str(obj) for obj in [obj])
+    if isinstance(obj, types.GeneratorType):
+        return list(obj)
+    raise TypeError ("Type %s not serializable" % type(obj))
+
+def use_jq_to_parse(data, jq_filter):
```

Comment:
not sure about the need to wrap `transform_with_jq` also worried about logging in `error` level raw customers data, I would rather not do this

Reply:
we are deleting this wrapping function